kasa/experimental/sslaestransport.py

```diff
@@ -490,5 +490,5 @@ def from_int(value: int) -> SmartErrorCode:
 ]
 
 SMART_AUTHENTICATION_ERRORS = [
-    SmartErrorCode.INVALID_ARGUMENTS,
+    SmartErrorCode.HOMEKIT_LOGIN_FAIL,
```

Comment:
`INVALID_ARGUMENTS` is no longer relevant / not relevant for this transport at all?

Reply:
`INVALID_ARGUMENTS` is a valid error code for this transport but it does not specifically refer to an authentication error. We may need to treat it separately during the login process.

Reply:
Oh, okay.. So I suppose this PR is ready to be merged?

Reply:
Yes, it was tested this morning with the H200 and all works ok except for an unclosed client session with I've now fixed in [`2203983` (#1169)](https://github.com/python-kasa/python-kasa/pull/1169/commits/220398374e34749378d5dd542cd3b2e899244454)

Reply:
Okay, cool! Feel free to merge as you wish.